libs/sync/src/lib/upload/select-files/select-files.component.ts

```diff
@@ -138,8 +138,8 @@ export class SelectFilesComponent implements AfterViewInit {
         // Add empty item just to trigger sitemap synchronization
         this.selection.setSelection({
           uuid: '',
-          title: '',
-          originalId: '',
+          title: 'Sitemap',
+          originalId: 'SITEMAP',
```

Comment:
should the desktop app version number be increased ?

Reply:
no, that's for the synchronizer